tests/test-team/template-mgmt-component-tests/template-mgmt-create-nhs-app-template-page.component.ts

```diff
@@ -13,9 +14,20 @@ export const nhsAppNoTemplateSessionData: Session = {
   nhsAppTemplateMessage: '',
 };
 
+export const nhsAppNoTemplateSessionDataForInput: Session = {
+  __typename: 'SessionStorage',
+  id: '4d98b0c4-7777-0000-1111-95eb27590011',
+  createdAt: '2024-09-19T23:36:20.815Z',
+  updatedAt: '2024-09-19T23:36:20.815Z',
+  templateType: TemplateType.NHS_APP,
+  nhsAppTemplateName: '',
+  nhsAppTemplateMessage: '',
+};
+
 test.describe('Create NHS App Template Page', () => {
   const sessionStorageHelper = new SessionStorageHelper([
     nhsAppNoTemplateSessionData,
+    nhsAppNoTemplateSessionDataForInput,
```

Comment:
The names of these variables is confusing, I'm not sure what "for input" means.  Is there a better way to name these?

Reply:
emptySessionForTemplateCreation?